src/Nethermind/Nethermind.Facade/BlockchainBridge.cs

```diff
@@ -68,11 +66,11 @@ public BlockchainBridge(OverridableTxProcessingEnv processingEnv,
             ILogFinder? logFinder,
             ISpecProvider specProvider,
             IBlocksConfig blocksConfig,
-            bool isMining)
+            IMiningConfig miningConfig)
         {
             _processingEnv = processingEnv ?? throw new ArgumentNullException(nameof(processingEnv));
-            _blockTree = processingEnv.BlockTree;
-            _stateReader = processingEnv.StateReader;
+            _blockTree = blockTree;
+            _stateReader = stateReader;
```

Comment:
Change from the overridable state reader to the global state reader. This is likely a mistake.